en/docs/develop/components/api-proxy.md

```diff
@@ -37,13 +37,13 @@ The following diagram illustrates the process of deploying an API in Choreo to t
 
 ![Deploy API ](../../../assets/img/api-proxies/deploy-api-proxies.png){.cInlineImage-threeQuarter}
 
-Choreo runs a professional, enterprise-grade CI/CD process to deploy APIs to its runtime(data plane) clusters. Under the hood, Choreo's data plane runs on a Kubernetes stack, hence benefits from all its features such as auto-scaling, auto-healing, secret management, liveness, readiness checks, and so on.
+Choreo runs a professional, enterprise-grade CI/CD process to deploy APIs to its runtime(data plane) clusters. Under the hood, Choreo's data plane runs on a Kubernetes stack, hence benefits from all its features such as auto-scaling, auto-healing, secret management, liveness, readiness checks, and so on. This CI/CD pipeline needs to be manually managed by you via the Choreo console. 
```

Comment:
Suggestion for the last sentence: "You are required to manually manage this CI/CD pipeline via the Choreo Console."